src/VisualStudio/Core/Def/Implementation/Library/AbstractLibraryService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Library.VsNavInfo;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Library
+{
+    internal abstract class AbstractLibraryService : ILibraryService
+    {
+        public Guid LibraryId { get; }
+        public __SymbolToolLanguage PreferredLanguage { get; }
```

Comment:
I only found one non-COM "__" variable in Roslyn, and it seems to be named that way because the single-underscore version of the name was already taken for the related property: http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Intellisense/ModelComputation.cs,265f5ccd16f2de32

This introduces a new "__x" naming use case to the codebase -- is it intentional?